packages-generation/manifest-utils.psm1

```diff
@@ -67,7 +67,7 @@ function Build-VersionsManifest {
         
         $ltsStatus = Get-VersionLtsStatus -Version $versionKey -LtsRules $ltsRules
         $stable = $version.PreReleaseLabel ? $false : $true
-        [array]$releaseAssets = $release.assets | ForEach-Object { New-AssetItem -ReleaseAsset $_ -Configuration $Configuration }
+        [array]$releaseAssets = $release.assets | Where { $_.Name -ne "hashes.sha256" } | ForEach-Object { New-AssetItem -ReleaseAsset $_ -Configuration $Configuration }
```

Comment:
The unit test https://github.com/actions/versions-package-tools/blob/b964a9871b5d9c7ae9b11f790745ce9d0aa7e7dd/packages-generation/manifest-utils.Tests.ps1#L76 should be expanded in order to cover this condition